woocommerce/woocommerce-android
# Jetpack CP: fetch site's metadata from the remote site directly

Closes: #5137

### Description
This PR updates FluxC to bring the changes that were discussed in p91TBi-6lK-p2 

### Testing instructions
1. You need a Jetpack CP site for the test.
2. Open the app.
3. Login to your Jetpack CP site.
4. Open the site, and then update the site's title.
5. Re-open the app (you may need to force-close the app before opening it to force trigger the site fetch call)
6. Confirm that the site's title has been updated.
